code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -520,6 +520,16 @@
 	construction_time = 100
 	category = list("Exosuit Equipment")
 
+//NSV-related change
```

Comment:
This is okay for now, but in future we'd usually make our own copy of the file under our subdirectory for ease of use

Reply:
Think this is what you meant, if not let me know